### ISSUE TITLE
Possibility of specify different aria-label for Cards

### ISSUE THREAD

**Issue by sandrodw3** [liferay/clay#5529]
## Context
Hi guys! 😄 This is a follow-up of https://github.com/liferay/clay/issues/5425. 
In that issue, I asked for the possibility of adding an `aria-label` for all Cards component. As far as I could see, the fix has been adding as `aria-label` the same value we have for `title`. I think that in some cases, we need to add a different value for the `aria-label`, for example if we want to add extra info that the `title` does not have.

## What is your proposal?
For that reason, I think it would be ideal to be able to specify the `aria-label` as a prop, something like `titleAriaLabel`, so we can give it a different value. 

## Example
I'm giving our case as example. For this card, the title says "Styles from Theme", but for the `aria-label` we would like to have "Styles from Theme (marked as default)". We could achieve that if the `aria-label` itself was a prop for Card components, as I suggested.

![image](https://github.com/liferay/clay/assets/2444936/ade00c5a-1b97-44b8-afac-42914233af17)

Please let me know your thoughts, thanks!
